app/modules/node-app-service/main.tf

```diff
@@ -97,6 +97,43 @@ resource "azurerm_linux_web_app_slot" "staging" {
   }
 }
 
+resource "azurerm_app_service_certificate" "custom_hostname" {
+  count = var.custom_hostname_enabled ? 1 : 0
```

Comment:
Maybe we don't need this flag variable? Can we do a `var.custom_hostname != null` check instead?

Reply:
Fixed in [ec6468c](https://github.com/Planning-Inspectorate/infrastructure-environments/pull/80/commits/ec6468ccebb29a58feeec51e6ee65de9e51c126a)